app/views/about.html

```diff
@@ -19,6 +19,8 @@ <h2 id="version">Version</h2>
                   <dd>{{version.master.openshift || 'unknown'}}</dd>
                   <dt>Kubernetes Master:</dt>
                   <dd>{{version.master.kubernetes || 'unknown'}}</dd>
+                  <dt>OpenShift Web Console:</dt>
```

Comment:
Web Console or Management Console? What are we actually calling it in the docs

Reply:
Hm, I thought we had switched everything to "Web Console." At least it's what we call it in the customization doc.

https://docs.openshift.org/latest/install_config/web_console_customization.html

Reply:
Yeah looking through the docs it seems to consistently be "Web Console"